applications/nrf5340_audio/unicast_client/overlay-unicast_client.conf

```diff
@@ -16,6 +16,8 @@ CONFIG_BT_MAX_CONN=2
 CONFIG_BT_MAX_PAIRED=2
 CONFIG_BT_EXT_ADV=y
 
+CONFIG_BT_BUF_EVT_RX_COUNT=20
```

Comment:
Why 20?

Reply:
It doesn't as the defaults suffice in this case.